stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -4153,15 +4119,15 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorage {
   /// - parameter idealBucket: The ideal bucket for the element being deleted.
   /// - parameter offset: The offset of the element that will be deleted.
   /// Precondition: there should be an initialized entry at offset.
-  internal mutating func nativeDeleteImpl(
+  internal mutating func nativeDelete(
```

Comment:
To be clear, I am not opposed to the rename, but I prefer not to see renames as a part of other commits, because it makes it harder to evaluate changes.